src/core/qgsrendercontext.h

```diff
@@ -269,9 +288,39 @@ class CORE_EXPORT QgsRenderContext
 
     //setters
 
-    //! Sets coordinate transformation.
+    /**
+     * Sets the current coordinate transform for the context.
+     *
+     * This usually represents the coordinate transform required to transform a layer
+     * which is currently being rendered back to the CRS of a rendered map.
+     *
+     * Set to an invalid QgsCoordinateTransform to indicate that no transformation is required.
+     *
+     * \see coordinateTransform()
+     */
     void setCoordinateTransform( const QgsCoordinateTransform &t );
+
     void setMapToPixel( const QgsMapToPixel &mtp ) {mMapToPixel = mtp;}
+
+    /**
+     * Sets the map \a extent to be rendered.
+     *
+     * When rendering a map layer, this extent must be in the corresponding layer's CRS.
+     *
+     * \see extent()
+     */
```

Comment:
Is this comment also related to setExtent() ? maybe it should be merged with the comment block below as sipify does not seem to use this one...